### PR TITLE
Media Asset Page variant support

### LINKED ISSUE #3772: Variant support for Media Asset Pages
**Is your feature request related to a problem? Please describe.**
We need to support media asset pages on the services that have variants. 

```
/serbian/srbija-49427344/lat
/serbian/srbija-49427344/cyr
/ukchina/49375846/simp
/ukchina/49375846/trad
/zhongwen/chinese-news-49631219/simp
/zhongwen/chinese-news-49631219/trad
```

**Describe the solution you'd like**
Enable support for services that have variants. Change the getInitalData function to optionally include variants if they are valid. Also update the fixture data to work for services with variants for the media asset pages. 

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: The following routes should render when running simorgh locally on the PR. 

http://localhost.bbc.com:7080/serbian/srbija-49427344/lat
http://localhost.bbc.com:7080/serbian/srbija-49427344/cyr
http://localhost.bbc.com:7080/ukchina/49375846/simp
http://localhost.bbc.com:7080/ukchina/49375846/trad
http://localhost.bbc.com:7080/zhongwen/chinese-news-49631219/simp
http://localhost.bbc.com:7080/zhongwen/chinese-news-49631219/trad

- [x] This feature is expected to need manual testing.

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves #3772

**Overall change:** Adds support for service with variants for the Media Asset Pages

**Code changes:**

- Enable variant support in the regex, server, getInitalData and fixture data for the mediaAssetPages. 
- Unit tests for all changed logic

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

**Testing:**
With this branch checked out the following URLs should render a page correctly: 

http://localhost.bbc.com:7080/serbian/srbija-49427344/lat
http://localhost.bbc.com:7080/serbian/srbija-49427344/cyr
http://localhost.bbc.com:7080/ukchina/49375846/simp
http://localhost.bbc.com:7080/ukchina/49375846/trad
http://localhost.bbc.com:7080/zhongwen/chinese-news-49631219/simp
http://localhost.bbc.com:7080/zhongwen/chinese-news-49631219/trad

### CODE REVIEW

**Review by dr3 — CHANGES_REQUESTED** [review]
Cyyyyyypprreesssssss

NB: were currently only testing `simp`

**Comment by pjlee11** [comment]
@dr3 the Cypress tests for MediaAssetPage on all services that support variants is `mediaAssetPage: { path: undefined, smoke: false },`

**Review by dr3 — APPROVED** [review]

File: src/app/routes/getInitialData/mediaAssetPage/index.test.js [hunk 1]
```diff
@@ -12,20 +12,55 @@ baseUrl.mockImplementation(() => mockBaseUrl);
 jest.mock('../utils/fetchData', () => jest.fn());
 fetchData.mockImplementation(() => mockData);
 
+const defaultServiceParam = 'pidgin';
```
Comment (ryanmccombe) [inline src/app/routes/getInitialData/mediaAssetPage/index.test.js:15]: don't think we use these variables, other than to move them into an object later?

Could just create the object here

**Review by ryanmccombe — APPROVED** [review]

**Review by jroebu14 — APPROVED** [review]
LGTM

**Pull request merged**